### PR TITLE
- gcemetadata

### PR DESCRIPTION
**Pull request by rjschwei**
+ Support setting the format for the identity token, supported
    values are "standard" or "full" per the API spec.

### CODE REVIEW

**Review by jgleissner — APPROVED** [review]

File: gcemetadata/gcemetadata [hunk 1]
```diff
@@ -115,9 +115,7 @@ def main():
         getopt_metaopts.append('diskid=')
         getopt_metaopts.append('licenseid=')
         getopt_metaopts.append('netid=')
-        getopt_metaopts.append('output=')
         getopt_metaopts.append('query=')
```
Comment (smarlowucf) [inline gcemetadata/gcemetadata:118]: Did these accidentally get removed? (output, and xml)

File: gcemetadata/gcemetadata [hunk 2]
```diff
@@ -180,6 +178,14 @@ def main():
...
                 sys.exit(1)
             meta.set_identity_arg(val)
+        elif opt == '--identity-format':
+            supported_vals = ['full', 'standard']
+            if val not in supported_vals:
+                msg = '--identity-format possible values: "%s"' %supported_vals
+                print >> sys.stderr, msg
+                sys.exit(1)
+            meta.set_identity_format(val)
+            continue
```
Comment (smarlowucf) [inline gcemetadata/gcemetadata:188]: Unnecessary continue

File: gcemetadata/gcemetadata (hunk 1 above)
Comment (rjschwei) [reply gcemetadata/gcemetadata:118]: No they are in the general options list, which gets added, thus these were duplicate entries all along

File: gcemetadata/gcemetadata (hunk 2 above)
Comment (rjschwei) [reply gcemetadata/gcemetadata:188]: No the continue is necessary of the --identity-format which is not an endpoint url will be added to the options passed for query and thus "unavailable" will be printed.

**Pull request merged**